pkgs/desktops/gnome-3/extensions/dash-to-panel/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, glib, gettext }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-dash-to-panel-${version}";
+  version = "v11";
```

Comment:
Minor thing: It's common practice in nix not to use a "v" before version numbers. You should move that to the `rev` attribute passed to `fetchFromGithub` as such:
`rev = "v${version}";`.